src/main/scala/com/snowflake/snowpark/functions.scala

```diff
@@ -3887,6 +3887,256 @@ object functions {
+   *  0      | 0           |
+   *  2      | 1.25992105  |
+   *  -10    | -2.15443469 |
+   *  [NULL] | [NULL]      |
+   *  --------+-------------+
+   *
+   * @since 1.15.0
+   * @param column Column to calculate the cubic root.
+   * @return Column object.
+   */
+  def cbrt(columnName: String): Column = {
+    cbrt(col(columnName))
+  }
+
+  /**
+   * This function converts a JSON string to a variant in Snowflake.
+   *
+   * In Snowflake the values are converted automatically, however they're converted as variants,
+   * meaning that the printSchema
+   * function would return different datatypes.
+   * To convert the datatype and it to be printed as the expected datatype,
+   * it should be read on the
+   * selectExpr function as "json['relative']['age']::integer"
+   * val data_for_json = Seq(
+   *   (1, "{\"id\": 172319, \"age\": 41, \"relative\": {\"id\": 885471, \"age\": 29}}")
+   *   (2, "{\"id\": 532161, \"age\": 17, \"relative\":{\"id\": 873513, \"age\": 47}}")
+   * )
+   * val data_for_json_column = Seq("col1", "col2")
+   * val df_for_json = session.createDataFrame(data_for_json).toDF(data_for_json_column)
+   *
+   * val json_df = df_for_json.select(
+   *   from_json(col("col2")).as("json")
+   * )
+   *
+   * json_df.selectExpr(
+   *   "json['id']::integer as id"
+   *   , "json['age']::integer as age"
+   *   , "json['relative']['id']::integer as rel_id"
+   *   , "json['relative']['age']::integer as rel_age"
+   * ).show(10, 10000)
+   * </pre>
+   *
+   * <pre>
+   * -----------------------------------------
+   * |"ID"    |"AGE"  |"REL_ID"  |"REL_AGE"  |
+   * -----------------------------------------
+   * |172319  |41     |885471    |29         |
+   * |532161  |17     |873513    |47         |
+   * -----------------------------------------
+   * </pre>
+   * @since 1.15.0
+   * @param e String column to convert to variant.
+   * @return Column object.
+   */
+  def from_json(e: Column): Column = {
+    builtin("TRY_PARSE_JSON")(e)
+  }
+
+  /**
+   * Returns the value of sourceExpr cast to data type
+   * targetType if possible, or NULL if not possible.
+   * Example::
+   *
+   * df = session.create_dataframe(['0.12', 'USD 27.90',
+   * '13.97 USD', '97.0', '17,-'], schema=["a"])
+   * df.select(try_cast(col("a"), FloatType()).as_('ans')).collect()
+   * [Row(ANS=0.12), Row(ANS=None), Row(ANS=None), Row(ANS=None), Row(ANS=None)]
+   * @since 1.15.0
+   * @param source Any castable expression
+   * @param Target The type of the result
+   * @return The result is of type targetType.
+   *  special version of CAST for a subset of datatype conversions.
+   * It performs the same operation
+   * (i.e. converts a value of one data type into another data type),
+   * but returns a NULL value instead of raising an error
+   * when the conversion can not be performed.
+   * The column argument must be a string column in Snowflake.
+   */
+  def try_cast(e: Column, targetType: DataType): Column = {
+    e.cast(targetType)
```

Comment:
Do you have tests cases to validate that this returns null if cast fails? I think this might fail and that you might need to use try_cast built int